lib/renderer.coffee

```diff
@@ -14,17 +14,25 @@ highlighter = null
 packagePath = path.dirname(__dirname)
 
 exports.toHtml = (text='', filePath) ->
+  render text, filePath
+    .then (html) ->
+      sanitize html
+    .then (html) ->
+      resolveImagePaths html, filePath
+    .then (html) ->
+      tokenizeCodeBlocks html
```

Comment:
`tokenizeCodeBlocks` should be removed since we are now using Highlight.js to highlight code no ?
https://github.com/asciidoctor/atom-asciidoc-preview/blob/master/lib/renderer.coffee#L80